packages/dd-trace/src/appsec/iast/analyzers/analyzers.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
-  'WEAK_HASH_ANALYZER': require('./weak-hash-analyzer')
+  'WEAK_HASH_ANALYZER': require('./weak-hash-analyzer'),
```

Comment:
In other files with multiple exports, they are ordered by name, `WEAK_CIPHER_ANALYZER` before `WEAK_HASH_ANALYZER`